### PR TITLE
[Network / Receive] Clean 2008 ragexeRE files

### PR DESCRIPTION
**Pull request by alisonrag**
RagexeRE 2008 files

### CODE REVIEW

File: src/Network/Receive/kRO/RagexeRE_2008_09_10a.pm [hunk 1]
```diff
@@ -22,28 +22,7 @@ use base qw(Network::Receive::kRO::RagexeRE_2008_08_27a);
...
 
-1;
-
-=pod
-//2008-09-10aRagexeRE
-packet_ver: 24
-0x0436,19,wanttoconnection,2:6:10:14:18
-0x0437,7,actionrequest,2:6
-0x0438,10,useskilltoid,2:4:6
-0x0439,8,useitem,2:4
```
Comment (sctnightcore) [inline src/Network/Receive/kRO/RagexeRE_2008_09_10a.pm]: dont del it !

Comment (lututui) [reply src/Network/Receive/kRO/RagexeRE_2008_09_10a.pm]: Until we have a better way to document when a packet was added, it's better to keep it there

Comment (lututui) [reply src/Network/Receive/kRO/RagexeRE_2008_09_10a.pm]: Is that up to date?
If so, I'm OK with deleting it from the Network src. Also, maybe we should leave a note to lookup that file?

**Comment by Nipodemos** [comment]
since the packetdescription is up to date, i believe that this pull request can be merged

**Comment by steadybear69** [comment]
do you have updated packetlist?

**Pull request merged**